pkg/registry/apis/provisioning/migrate.go

```diff
@@ -68,7 +68,7 @@ func (c *migrateConnector) Connect(
 		job, err := c.jobs.Insert(ctx, &provisioning.Job{
 			ObjectMeta: v1.ObjectMeta{
 				Namespace: cfg.Namespace,
-				Name:      fmt.Sprintf("%s:%s", cfg.Name, provisioning.JobActionMigrate),
+				Name:      fmt.Sprintf("%s-%s", cfg.Name, provisioning.JobActionMigrate),
```

Comment:
🤔  for sync+migrate... should they both use the same name?  that would avoid them ever running together

Reply:
That makes sense to me!